### PR TITLE
main/raspberrypi: upgrade to 0.20190416

### PR DESCRIPTION
**Pull request by innerspacepilot**
Updated version

### CODE REVIEW

File: main/raspberrypi/APKBUILD [hunk 1]
```diff
@@ -44,6 +44,7 @@ package() {
 	# nuke the unwanted stuff
 	rm -rf "$pkgdir"/opt/vc/src
 	rm -rf "$pkgdir"/opt/vc/lib/*.a
+	cp -r "$pkgdir"/opt/vc/etc "$pkgdir"
```
Comment (fabled) [inline main/raspberrypi/APKBUILD]: I don't understand why the etc config is copied. Does it fix something? Should it then be moved instead?

Comment (innerspacepilot) [reply main/raspberrypi/APKBUILD]: This is the reason https://github.com/raspberrypi/userland/commit/e5803f2c986cbf8c919c60278b3231dcdf4271a6

Comment (innerspacepilot) [reply main/raspberrypi/APKBUILD]: Seems like moving it would be the right way.
I'll make a change

**Comment by innerspacepilot** [comment]
fixed

**Comment by innerspacepilot** [comment]
@fabled, Is it ok now?

**Pull request closed without merging**